pom.xml

```diff
@@ -1620,6 +1624,67 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>enforce-jdk11</id>
+            <activation>
+                <jdk>11</jdk>
```

Comment:
I'm not sure if this has to be a profile (@carlspring ping), but if it does - then it needs to be activated by default. Nobody will ever know to enable it when they build the project for the first time. :)

Reply:
Well, doesn't:
```
            <activation>
                <jdk>11</jdk>
            </activation>
```
do exactly that? It will get triggered if the JDK is 11.

@AricMonary : Have you tested how this behaves with JDK8 and JDK11? Could you please confirm the behaviour with example command + outputs? Thanks! :)

Reply:
@carlspring See previous post for results.